postgresql/gen-yaml/gen.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o nounset
+
+mkdir -p manifests/${NS}/${NAME}/upstream
+
+# Fetch from helm repo: https://github.com/bitnami/charts/tree/main/bitnami/postgresql
+helm repo add bitnami https://charts.bitnami.com/bitnami
+
+helm template "${NAME}" bitnami/postgresql --version "${BITNAMI_POSTGRES_RELEASE}" \
```

Comment:
I think it would be worth to add `metrics.enabled=true` in order to enable Prometheus metrics collection